db/migrate/20210621234939_backfill_email_log_topic_id.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class BackfillEmailLogTopicId < ActiveRecord::Migration[6.1]
```

Comment:
We need to do this in a post migration since we need the updated code to be deployed before running this migration to ensure we are backfilling correctly.

Reply:
Dang it I second guessed myself here...will switch to a post migrate